### ISSUE TITLE
Possible bug: PCALoss.empirical_epsilon_percentile default is 0.99, but EmpiricalEpsilon uses np.nanpercentile (0–100 scale)

### ISSUE THREAD

**Issue by twheatcroft07** [danbider/lightning-pose#380]
Hi — I think there may be a bug or at least a confusing default around the empirical epsilon used for PCA losses.

### Summary

PCALoss appears to default to:

`empirical_epsilon_percentile: float = 0.99
`
but the empirical epsilon is ultimately computed with:

`np.nanpercentile(loss, self.percentile, axis=0)
`
which expects a percentile on the 0–100 scale, not the 0–1 scale.

That means the current default may be using the 0.99th percentile instead of the likely intended 99th percentile.

### Relevant code

In losses.py, PCALoss has:
```

class PCALoss(Loss):
    def __init__(
        self,
        loss_name: Literal["pca_singleview", "pca_multiview"],
        components_to_keep: int | float = 0.95,
        empirical_epsilon_percentile: float = 0.99,
        epsilon: float | None = None,
        empirical_epsilon_multiplier: float = 1.0,
        ...
    ):
        ...
        self.pca = KeypointPCA(
            loss_type=self.loss_name,
            data_module=data_module,
            components_to_keep=components_to_keep,
            empirical_epsilon_percentile=empirical_epsilon_percentile,
            ...
        )

```
and later:

```
if epsilon is not None:
    self.epsilon = torch.tensor(epsilon, dtype=torch.float, device=self.device)
else:
    self.epsilon = self.pca.parameters["epsilon"] * empirical_epsilon_multiplier

```
In pca.py, KeypointPCA._set_parameter_dict() does:

```
self.parameters["epsilon"] = EmpiricalEpsilon(
    percentile=self.empirical_epsilon_percentile
)(loss=self.compute_reprojection_error())

```
In helpers.py, EmpiricalEpsilon.__call__() does:

`return np.nanpercentile(flattened_loss, self.percentile, axis=0)
`
### Why this seems problematic

np.nanpercentile expects percentile values in [0, 100].

So with the current PCALoss default:

empirical_epsilon_percentile = 0.99

the empirical epsilon would be set at the 0.99th percentile of the reprojection error distribution, which is extremely small.

That would make the epsilon-insensitive PCA loss behave almost like:

relu(loss - very_small_number)

so the PCA loss is effectively active on nearly all samples, rather than only on unusually bad reprojection errors.

### Possible intended behavior

I suspect the intended default may have been something like:

99.0 instead of 0.99

or alternatively EmpiricalEpsilon could be expecting fractions and internally multiplying by 100 before calling np.nanpercentile.

### Suggested fixes

One of these seems appropriate:

Change the PCALoss default from 0.99 to 99.0

Clarify in the docs/configs that empirical_epsilon_percentile is on a 0–100 scale

Add validation, e.g. warning/error when 0 < empirical_epsilon_percentile <= 1

Standardize this parameter across the API, since KeypointPCA appears to use a different default style

### Question

Is this an actual bug, or is the intended convention that empirical_epsilon_percentile should be given on the 0–100 scale and PCALoss just has the wrong default?

Thanks.
